packages/zapp/console/src/components/WorkflowGraph/WorkflowGraph.tsx

```diff
@@ -1,111 +1,44 @@
-import { dNode } from 'models/Graph/types';
-import { Workflow } from 'models/Workflow/types';
 import * as React from 'react';
 import ReactFlowGraphComponent from 'components/flytegraph/ReactFlow/ReactFlowGraphComponent';
-import { Error } from 'models/Common/types';
 import { NonIdealState } from 'components/common/NonIdealState';
-import { DataError } from 'components/Errors/DataError';
-import { WaitForQuery } from 'components/common/WaitForQuery';
-import { useQuery } from 'react-query';
-import { makeNodeExecutionDynamicWorkflowQuery } from 'components/Workflow/workflowQueries';
-import { createDebugLogger } from 'common/log';
 import { CompiledNode } from 'models/Node/types';
 import { TaskExecutionPhase } from 'models/Execution/enums';
-import { NodeExecutionsByIdContext } from 'components/Executions/contexts';
-import { useContext } from 'react';
-import { checkForDynamicExecutions } from 'components/common/utils';
-import { transformerWorkflowToDag } from './transformerWorkflowToDag';
 
 export interface WorkflowGraphProps {
   onNodeSelectionChanged: (selectedNodes: string[]) => void;
   onPhaseSelectionChanged: (phase: TaskExecutionPhase) => void;
   selectedPhase?: TaskExecutionPhase;
   isDetailsTabClosed: boolean;
-  workflow: Workflow;
+  mergedDag: any;
+  error: any;
```

Comment:
can we update type here?
```suggestion
  error: Error | null;
```

Reply:
fixed